tools/src/main/java/org/apache/kafka/trogdor/workload/SustainedConnectionSpec.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.trogdor.workload;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.kafka.trogdor.task.TaskController;
+import org.apache.kafka.trogdor.task.TaskSpec;
+import org.apache.kafka.trogdor.task.TaskWorker;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class SustainedConnectionSpec extends TaskSpec {
```

Comment:
Could we please add a javadoc with a sample JSON config for this? Similar to what we have for `ProduceBenchSpec` - I find those examples very valuable.

It would be good to explain non-obvious parameters like `refreshRateMs` here too. I see we have comments in the SustainedConnectionWorker like `// This variable is used to maintain the scheduling of the connection keep-alive.` - those comments are more useful in the spec class as that's what you generally need to read when you want to use the worker

Reply:
Done.